apollo-runtime/src/main/java/com/apollographql/apollo/internal/RealApolloStore.kt

```diff
@@ -23,11 +23,15 @@ import java.util.UUID
 import java.util.WeakHashMap
 import java.util.concurrent.locks.ReentrantReadWriteLock
 
-class RealApolloStore(normalizedCache: NormalizedCache,
-                      private val cacheKeyResolver: CacheKeyResolver,
-                      val customScalarAdapters: CustomScalarAdapters,
-                      val logger: ApolloLogger) : ApolloStore, ReadableStore, WriteableStore {
+class RealApolloStore(
+    normalizedCache: NormalizedCache,
+    private val cacheKeyResolver: CacheKeyResolver,
+    val customScalarAdapters: CustomScalarAdapters,
+    val logger: ApolloLogger
+) : ApolloStore, ReadableStore, WriteableStore {
+
   private val optimisticCache = OptimisticNormalizedCache().chain(normalizedCache) as OptimisticNormalizedCache
+
   private val lock = ReentrantReadWriteLock()
   private val subscribers: MutableSet<RecordChangeSubscriber> = Collections.newSetFromMap(WeakHashMap())
```

Comment:
Ah, this one file might require more work! I'm not really sure we need the Weak reference stuff there but we can discuss that later.